packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -884,7 +890,7 @@ export class WalletService implements IWalletService {
 
     // this.logi('Notification', type);
 
-    cb = cb || function() {};
+    cb = cb || function () { };
```

Comment:
A space before the () has not been our lint convention (except before an `async` keyword)